osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@nHirsenkorn I still see an inconsistency because (as the name of the vector indicates) only Ego vehicles provide a vector **relative_ego_origin**, i.e. for non Ego vehicles there is less information on the interface.

In my opinion we need to agree on a vehicle reference point for all vehicles in the simulation, which not necessarily needs to be the rear axis but also not necessarily needs to be the center of the bounding box.

By the way, the real bounding box changes, even during runtime (opening doors but also vehicle movement, spring compression, etc.).